openmetadata-ui/src/main/resources/ui/src/components/Widgets/FeedsWidget/FeedsWidget.component.tsx

```diff
@@ -131,19 +136,20 @@ const FeedsWidget = () => {
                 {countBadge}
               </>
             ),
-            key: 'tasks',
+            key: ActivityFeedTabs.TASKS,
             children: (
               <ActivityFeedListV1
                 emptyPlaceholderText={t('message.no-tasks-assigned')}
                 feedList={threads}
                 hidePopover={false}
                 isLoading={loading}
                 showThread={false}
+                tab={ActivityFeedTabs.TASKS}
               />
             ),
           },
         ]}
-        onChange={(key) => setActiveTab(key)}
+        onChange={(key) => setActiveTab(key as ActivityFeedTabs)}
```

Comment:
Can we define a function outside the jsx? as this will create new function for each render.